packages/waitlist/package.json

```diff
@@ -4,7 +4,7 @@
   "type": "module",
   "devDependencies": {},
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": ""
```

Comment:
we need a "test" command but don't want to show any output when the package tests get run